string uri
int8 back_emf_state
---
string uri
int8 back_emf_state